src/BluetoothHciSocket.cpp

```diff
@@ -184,6 +187,17 @@ void BluetoothHciSocket::bindControl() {
   bind(this->_socket, (struct sockaddr *) &a, sizeof(a));
 }
 
+void BluetoothHciSocket::bindUser() {
+  struct sockaddr_hci a;
+
+  memset(&a, 0, sizeof(a));
+  a.hci_family = AF_BLUETOOTH;
+  a.hci_dev = 0;
```

Comment:
Does the dev number have to be zero or is based on the device id?

```bindRaw``` currently checks for the first device that is up or accepts an optional argument to specify a device.